examples/typescript/.sauce/config.yml

```diff
@@ -16,7 +16,15 @@ docker:
   # Affects how test files are transferred to the docker container when using the docker mode.
   fileTransfer: copy # Choose between mount|copy.
 cypress:
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
should be fixed here:)

Reply:
Grrr. I feel like I've fixed that 3 times! Thanks, @tianfeng92